### PR TITLE
Block Reward Increase with scheduled decrease to 1ETH BR - 3368.md

### PR DESCRIPTION
**Pull request by bitsbetrippin**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-3333.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+eip: 3333
+title: Increase block rewards to 3 ETH, with 2 Year Decay to 1 ETH Scheduled
+author: Michael D. Carter (BitsBeTrippin)
```
Comment (MicahZoltu) [inline EIPS/eip-3333.md:4]:
```suggestion
author: Michael D. Carter (@bitsbetrippin)
```

File: EIPS/eip-3333.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+eip: 3333
```
Comment (MicahZoltu) [inline EIPS/eip-3333.md:2]:
```suggestion
eip: 3368
```

File: EIPS/eip-3333.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+title: Increase block rewards to 3 ETH, with 2 Year Decay to 1 ETH Scheduled
+author: Michael D. Carter (BitsBeTrippin)
+discussions-to: https://discord.gg/3sjBbWKM4E (ETH R&D 1559-mining)
```
Comment (MicahZoltu) [inline EIPS/eip-3333.md:5]: Discussions-to links need to be to a semi-permanent asynchronous discussion forum.  A thread at https://ethereum-magicians.org/ is strongly preferred, though a new issue in this GitHub repository is considered an acceptable alternative if you unable or unwilling to create a thread over there.

File: EIPS/eip-3333.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+---
+eip: 3333
+title: Increase block rewards to 3 ETH, with 2 Year Decay to 1 ETH Scheduled
```
Comment (MicahZoltu) [inline EIPS/eip-3333.md:3]:
```suggestion
title: Set block reward to 3 ETH with a 2 Year Decay to 1 ETH
```
Try to phrase things throughout the EIP in the context of a future reader who may not have a knowledge of historical state, and also so they can be implemented without needing to know previous state (e.g., this could be implemented on a chain that currently has a 5 ETH block reward or a chain with a 1 ETH block reward).

File: EIPS/eip-3333.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+## Motivation
+With the passing of EIP-1559, the potential steep reduction of Ethereum Miner fee's, which currently are averaging a doubling of the exiting block reward have several scenerios where it can become the 50% key factor on keeping ethereum mining participation growing the current all time high hashpower, all the way to the London hard fork release. There are several scenerios where this can create a steep hashrate decrease when the BASEFEE starts to get burned vs distributed to miners. This can create an exteciential risk to ethereum's network by incentivizing miners to move to more profitable solutions, including broker services such as Nicehash and Miningrigrentals. To ward off this threat, a recommend action is to temporarly increase Eth Mining Rewards by +1 offset then began to decrease -2 Eth over 8 quarters, 2 full years from London's Hardfork. This should align well with the current optimisitic ETH 2.0 PoW/PoS handoff and appropriately set Eth Inflation on a managed reduction against a schedule that can be planned for. Additionally this provides a higher incentive for price discovery which can offset any further steep impacts for the mining security budget.
```
Comment (MicahZoltu) [inline EIPS/eip-3333.md:19]: Try to avoid mentioning London or other things related to mainnet or current state of Ethereum in EIPs.  Consider that an EIP may be implemented at any time in the future.  Perhaps reword this to say something along the lines of:

> A sudden drop in PoW mining rewards could result in a sudden precipitous decrease in mining profitability that may drive miners to auction off their hashrate to the highest bidder while they figure out what to do with their now "worthless" hardware.  If enough hashrate is auctioned off in this way at the same time, an attacker will be able to rent a large amount of hashing power for a short period of time at relatively low cost and potentially attack Ethereum.  By setting the block reward to X (where X is enough to offset the sudden profitability decrease) and then decreasing it over time to Y (where Y is a number below the sudden profitability decrease), we both avoid introducing long term inflation while at the same time spreading out the rate that individual miners cross into the unprofitable range.  This allows miners time to sell off/repurpose their hardware rather than a large amount of hashing power being put up for auction at the same time while people try to figure out what to do with their now "worthless" hardware.

File: EIPS/eip-3333.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    new_uncle_reward = (8 - k) * new_block_reward / 8
+
+This is the existing formula for uncle rewards, simply adjusted with `new_block_reward`.
+
+The nephew reward for `block.number >= FORK_BLKNUM` is
+
+    new_nephew_reward = new_block_reward / 32
+
+This is the existing formula for nephew rewards, simply adjusted with `new_block_reward`.
+
```
Comment (MicahZoltu) [inline EIPS/eip-3333.md:40]:
````suggestion
## Specification
### Constants
* `TRANSITION_START_BLOCK_NUMBER: TBD`
* `TRANSITION_DURATION: 8_000_000` (about two years)
* `TRANSITION_END_BLOCK_NUMBER: FORK_BLOCK_NUMBER + TRANSITION_DURATION`
* `STARTING_REWARD: 3_000_000_000_000_000_000`
* `ENDING_REWARD: 1_000_000_000_000_000_000`
* `REWARD_DELTA: STARTING_REWARD - ENDING_REWARD`

### Block Reward
```py
if block.number >= TRANSITION_END_BLOCK_NUMBER:
    block_reward = ENDING_REWARD
elif block.number >= TRANSITION_START_BLOCK_NUMBER:
    block_reward = STARTING_REWARD - REWARD_DELTA * TRANSITION_DURATION / (block.number - TRANSITION_START_BLOCK_NUMBER)
```
````
A smooth transition is generally preferable to cliffs, for the exact reasons outlined in the Motivation section.  😄 

Also, no need to mention uncle or nephew rewards since those formulas aren't changing.

File: EIPS/eip-3333.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+## Rationale
+A 3 ETH base reward with quarterly schedule on decrease. 0.25 reduction per 3 months for 2 years to a total reduction of -2 Eth.
```
Comment (MicahZoltu) [inline EIPS/eip-3333.md:42]: The Rational section is for discussing why specific decisions were made.  Consider writing a **very** brief statement about why 2 years was chosen as the transition duration here.
```suggestion
2 years was chosen because it gives miners sufficient time to find alternative uses for their hardware and/or get their hardware back out onto the open market (e.g., in the form of gaming GPUs) without flooding the market suddenly.
```

File: EIPS/eip-3333.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+type: Standards Track
+category: Core
+created: 2021-03-11
+---
+
+## Simple Summary
+Changes the block reward paid to proof-of-work (POW) miners to 3 ETH from existing 2 ETH, but kicks off a 0.25 quarterly drop to a final 1.0 ETH Reward 2 Years Later.
+
+## Abstract
+Starting with `FORK_BLKNUM` block rewards will be increased to a base of 3 ETH, uncle and nephew rewards will be adjusted accordingly. Every 3 months after, measured by EPOCH count, Block Reward drops 0.25 every quarter till Ethereum Block Reward reaches 1.0 BR per Block.
```
Comment (MicahZoltu) [inline EIPS/eip-3333.md:16]: See comments below about making this smooth rather than having cliffs every 3 months.  If that change is accepted, this should be updated to reflect that.

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
I'm pretty sure you don't want to delete EIP 3143 in this pull request, you'll need to un-delete that before this is merged.

The changes to the frontmatter are necessary before this can be merged as a draft, the rest is suggestions/feedback to help improve this EIP but can be applied later in the process if desired (or not at all if the author desires).

File: EIPS/eip-3333.md (hunk 2 above)
Comment (MicahZoltu) [reply EIPS/eip-3333.md:2]: Note: filename needs to be updated as well to reflect this.

File: EIPS/eip-3333.md (hunk 7 above)
Comment (bitsbetrippin) [reply EIPS/eip-3333.md:38]: accepted

File: EIPS/eip-3333.md (hunk 5 above)
Comment (bitsbetrippin) [reply EIPS/eip-3333.md:19]: Good add

File: EIPS/eip-3333.md (hunk 8 above)
Comment (bitsbetrippin) [reply EIPS/eip-3333.md:16]: Accepted, makes more sense

File: EIPS/eip-3333.md (hunk 3 above)
Comment (bitsbetrippin) [reply EIPS/eip-3333.md:5]: Will add

File: EIPS/eip-3333.md (hunk 4 above)
Comment (bitsbetrippin) [reply EIPS/eip-3333.md:3]: Copy, good change

File: EIPS/eip-3143.md [hunk 9]
```diff
@@ -1,53 +0,0 @@
...
-A 5 ETH base reward was chosen as a middle ground between wanting to prevent too high of an inflation rate (10.4% per annum for the first year at 5 ETH per block) and converting as many POW miners as possible into POS validators by making it easier to amass the required ETH needed through POW mining.
-
-## Backwards Compatibility
-There are no known backward compatibility issues with the introduction of this EIP.
-
-## Security Considerations
-There are no known security issues presented by this change.
-
-## Copyright
-Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (bitsbetrippin) [inline EIPS/eip-3143.md:53]: Restore

**Review by bitsbetrippin — COMMENTED** [review]
Accepted

**Comment by MicahZoltu** [comment]
Replaced by #3370
